GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1483,6 +1486,8 @@ bool IsCommitMessageValid()
                         try
                         {
                             if (!Regex.IsMatch(Message.Text, AppSettings.CommitValidationRegEx) &&
+                                Message.Text.StartsWith(fixupPrefix) &&
+                                Message.Text.StartsWith(squashPrefix) &&
```

Comment:
`Message.Text` will never begin with both of these prefixes. I think there's a logic bug here.

Reply:
Yes. Fixed. Thanks.